GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -75,15 +75,17 @@ - (void)setDataObject:(id<GDTCOREventDataObject>)dataObject {
   }
 }
 
-- (void)setFileURL:(NSURL *)fileURL {
-  if (fileURL != _fileURL) {
-    _fileURL = fileURL;
-  }
-}
-
-- (void)clearDataObjectTransportBytes {
-  if (_dataObjectTransportBytes != nil) {
-    _dataObjectTransportBytes = nil;
+- (void)writeToURL:(NSURL *)fileURL error:(NSError **)error {
+  if (_dataObjectTransportBytes) {
```

Comment:
I'd get rid of the dataObjectTransportBytes property and just call [dataObject transportBytes] here